USING.md

```diff
@@ -1,116 +1,216 @@
+options `--raw` and `json`, respectively.
 
-    $ zonemaster-cli --stop_level NOTICE example.com
 
-This will halt execution as soon as a NOTICE or higher message is received.
+## Translation
+
+### In Docker
+
+By default all messages are in English. By using the `--locale=LANG` option
+another language can be selected. Select "LANG" from the table below to have
+Zonemaster translated into that language.
+
+LANG | Language
+-----|---------
+da   | Danish
+en   | English
+fi   | Finnish
+fr   | French
+nb   | Norwegian
+es   | Spanish
+sv   | Swedish
+
+E.g.:
+```sh
+docker run -t --rm zonemaster/cli zonemaster.net --locale=da
+```
+
+An alternative is to set the `LC_ALL` environment variable with correct language
+value when the command is invoked, which can be useful if a shell alias is
+created. E.g.
+```sh
+docker run -e LC_ALL=da -t --rm zonemaster/cli zonemaster.net
+```
+
+### In local installation
+
+By default all messages are in the language set in the local environment (if
+available in Zonemaster) or else in English. By using the `--locale=LOCALE`
+option another language can be selected. Select "LOCALE" from the table below to
+have Zonemaster translated into that language.
+
+LOCALE      | Language
+------------|---------
+da_DK.UTF-8 | Danish
+en_US.UTF-8 | English
+fi_FI.UTF-8 | Finnish
+fr_FR.UTF-8 | French
+nb_NO.UTF-8 | Norwegian
+es-CL.UTF-8 | Spanish
+sv_SE.UTF-8 | Swedish
+
+E.g.:
+```sh
+docker run -t --rm zonemaster/cli zonemaster.net --locale=da_DK.UTF-8
+```
+
+If the environment variable `LANGUAGE` is set with correct LOCALE then no option
+is needed, e.g. `LANGUAGE=da_DK.UTF-8`. `zonemaster-cli` also respects `LC_ALL`,
+`LC_MESSAGES` and `LANG`. `LANGUAGE` takes precedence over the other, and then
+the order is `LC_ALL`, `LC_MESSAGES` and last `LANG`.
+
+## Advanced use
+
+There are some nice features available that can be of some use for advanced
+users.
 
-### Pre-delegation Testing
+### Only run specific test cases
 
-Before you do any delegation change at the level of the parent, either
-changing the NS records, glue address records or DS records, you might
-want to perform a check of your new child zone configuration so that
-everything you plan to change is in order. Zonemaster can do this for
-your, all you have to do is give Zonemaster all the parent data you
-plan to have for your new configuration. Any DNS lookups going for
-the parent will instead be answered by the data you entered.
+If you only want to run a specific test case rather than the whole suite of
+tests, you can do that as well. E.g. test only test case [Connectivity03]:
 
-    $ zonemaster-cli --ns ns1.example.com/192.168.23.23 \
-          --ns ns2.example.com/192.168.24.24 \
-          --ds 12345,3,1,123456789abcdef67890123456789abcdef67890
+[Connectivity03]:  https://github.com/zonemaster/zonemaster/blob/master/docs/specifications/tests/Connectivity-TP/connectivity03.md
```

Comment:
Why not put this link at the end of the file?

Reply:
My mistake. Thanks for catching it.